docs-site/docs/dist-train.md

```diff
@@ -20,9 +20,9 @@ the gradients of the weights, which are averaged via All-Reduce (provided by
 [NCCL](https://developer.nvidia.com/nccl)) for weight update following
 stochastic gradient descent algorithms (SGD).
 
-The all-reduce operation by NCCL can be used to reduce and synchronize the
+The All-reduce operation by NCCL can be used to reduce and synchronize the
```

Comment:
All-reduce, All-Reduce, all-reduce?
just be consistent.

Reply:
ok, I will change